samples/samples-js/GetProductsByName/function.json

```diff
@@ -0,0 +1,30 @@
+{
+  "bindings": [
+    {
+      "authLevel": "anonymous",
+      "name": "req",
+      "type": "httpTrigger",
+      "direction": "in",
+      "methods": [
+        "get",
+        "post"
+      ],
+      "route":"getproductsbyname/{name}"
```

Comment:
I don't see a C# function for this - is there a reason you're making a new one of this?

Reply:
before replicating all the functions from c# samples, I added this.

Reply:
Remove it for now then, if we think it's useful to add later we can (but should do it for all languages at that point)